src/fftvis/cpu/beams.py

```diff
@@ -90,8 +90,13 @@ def evaluate_beam(
 
     @staticmethod
     @nb.jit(nopython=True, parallel=False, nogil=False)
-    def get_apparent_flux_polarized(beam: np.ndarray, flux: np.ndarray):  # pragma: no cover
-        """Calculate apparent flux of the sources."""
+    def get_apparent_flux_polarized_beam(beam: np.ndarray, flux: np.ndarray):  # pragma: no cover
+        """
+        Calculate apparent flux of the sources. Here we assume that the
```

Comment:
Sphinx auto-documentation requires you to have the first line of the docstring be a single sentence, and then to have a clear line before a potential extended description. I recommend following this format and separating the first sentence from the rest.

Reply:
Fixed this.